jetstream/workflows/run.yaml

```diff
@@ -77,3 +90,24 @@ spec:
         "--bucket={{workflow.parameters.bucket}}"
       ]
     activeDeadlineSeconds: 600   # terminate container template after 10 minutes
+
+  - name: analyse-and-export
+    inputs:
+      parameters:
+        - name: slug
+        - name: date
+    steps:
+    - - name: analyse-experiment
+        template: analyse-experiment  
+        arguments:
+          parameters:
+          - name: slug
+            value: "{{inputs.parameters.slug}}"
+          - name: date
+            value: "{{inputs.parameters.date}}"
+    - - name: export-statistics
+        template: export-statistics
+        arguments:
+          parameters:
+          - name: slug
+            value: "{{inputs.parameters.slug}}"
```

Comment:
One good thing that came out of this exercise is that I found a way to rewrite the Argo workflow to run the json export immediately after the analysis has been completed for an experiment. Currently, Argo is waiting for all experiment analyses to finish and then runs the json data export.

I'll open a separate PR for this.